LinkedListExample/LinkedListExample.Lib/IntegerLinkedList.cs

```diff
@@ -6,21 +6,230 @@
+        private IntegerNode _head;      // IntegerNode is a variable type 
 
-        public IntegerLinkedList(int v)
+        public IntegerLinkedList()
         {
-            this.v = v;
+            _head = null;           // 空的List，第一个加入空List，为第一个值
         }
 
-        public int Count { get; }
-        public int Sum { get;  }
+        public IntegerLinkedList(int v)    {
+            _head = new IntegerNode(v);
+           
+            /* this._v = v;     
+             * // "this."  refers    to the current instantiation of this class
+                             // if _v and v is different, so "this." is optional    */
+        }
+
+        public int Count {
+            get
+            {
+                // method: constructor in Class Integernode
+
+                if (_head == null)
+                    return 0;
+                else
+                    return _head.Count;     // see property in Integernode Class
+            } 
+        }
+        public int Sum {
+            get {
+                if (_head == null)
+                    return 0;               // nothing is before
+                else
+                    return _head.Sum;       // see property in Integernode Class
+            } 
+        }
+
+        public void Append(int v)   {
+
+            if (_head == null)
+                _head = new IntegerNode(v);
+            else
+                _head.Append(v);                    // jump to the property in IntegerNode
+        }
+
+        public void Remove(int v)   {
+
+            if (_head == null)
+                throw new IndexOutOfRangeException();
+            else
+                _head.Remove(v);                    // start checking from the 1st value => head
+        }                                           // 1st value can infer the values after
+
+        public bool Delete(int v)   {
+            if (_head == null)
+                return false;
+            else
+                return _head.Delete(v);             // start checking from the 1st value => head
+        }                                           // 1st value can infer the values after
+
+        public override string ToString()   {
+            string ListValueString = "{";
+            IntegerNode NodeNow = _head;
+            while (NodeNow != null)                 // when it's not the last one
+            {
+                ListValueString += NodeNow.ToString() + ";";    // conversion & format
+                NodeNow = NodeNow.Next;                         // Next: return _next
+            }
+            ListValueString = ListValueString.Substring(0, ListValueString.Length - 1) + "}";
+            return ListValueString;
+        }
+
+        public string RemoveDup()    {
+            IntegerNode CurrentOne = _head;
+            IntegerNode CompareNode = _head;
+            int ComparedValue = CompareNode.GetValue;
+
+            if (_head == null)
+                return "not cool ... u wrong";
```

Comment:
who is the comment for - the user?

Reply:
Sir for this one, yes. It's just a little joke. :)